test/integration/federation/crud_test.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package federation
+
+import (
+	"testing"
+
+	"github.com/pborman/uuid"
+
+	"k8s.io/kubernetes/test/integration/federation/framework"
+)
+
+// TestFederationCRUD validates create/read/update/delete operations for federated resource types.
+func TestFederationCRUD(t *testing.T) {
```

Comment:
What kind of failure are you worried about?  The tests run separately via [golang subtests] (https://blog.golang.org/subtests), they just share fixture.  Since each test only cares about resources it creates and isolates those resources in a unique namespace, there is very little chance of overlap.

Even if this test as proposed became a source of failure, modifying it would be relatively trivial.  The bulk of the work is done in the framework, this test just ties the test and the types together.